exchanges/kucoin/kucoin_test.go

```diff
@@ -2714,35 +2807,971 @@ func TestGetOpenInterest(t *testing.T) {
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestGetOrderInfoByOrderID(t *testing.T) {
+	t.Parallel()
+	_, err := ku.GetOCOOrderInfoByOrderID(context.Background(), "")
+	require.ErrorIs(t, err, order.ErrOrderIDNotSet)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.GetOCOOrderInfoByOrderID(context.Background(), "order-id-here")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestGetOrderInfoByClientOrderID(t *testing.T) {
+	t.Parallel()
+	_, err := ku.GetOCOOrderInfoByClientOrderID(context.Background(), "")
+	require.ErrorIs(t, err, order.ErrOrderIDNotSet)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.GetOCOOrderInfoByClientOrderID(context.Background(), "client-order-id-here")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestGetOrderDetailsByOrderID(t *testing.T) {
+	t.Parallel()
+	_, err := ku.GetOCOOrderDetailsByOrderID(context.Background(), "")
+	require.ErrorIs(t, err, order.ErrOrderIDNotSet)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.GetOCOOrderDetailsByOrderID(context.Background(), "order-id-here")
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestGetOCOOrderList(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.GetOCOOrderList(context.Background(), "0", "0", "BTC-USDT", time.Time{}, time.Now(), []string{})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestPlaceMarginHFOrder(t *testing.T) {
+	t.Parallel()
+	_, err := ku.PlaceMarginHFOrder(context.Background(), &PlaceMarginHFOrderParam{})
+	require.ErrorIs(t, err, common.ErrNilPointer)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.PlaceMarginHFOrder(context.Background(), &PlaceMarginHFOrderParam{
+		ClientOrderID:       "first-order",
+		Side:                "sell",
+		Symbol:              currency.Pair{Base: currency.BTC, Delimiter: currency.DashDelimiter, Quote: currency.ETH},
+		OrderType:           "market",
+		SelfTradePrevention: "CB",
+		Price:               1234,
+		Size:                0.0000001,
+	})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
+}
+
+func TestPlaceMarginHFOrderTest(t *testing.T) {
+	t.Parallel()
+	_, err := ku.PlaceMarginHFOrderTest(context.Background(), &PlaceMarginHFOrderParam{})
+	require.ErrorIs(t, err, common.ErrNilPointer)
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, ku)
+	result, err := ku.PlaceMarginHFOrderTest(context.Background(), &PlaceMarginHFOrderParam{
+		ClientOrderID:       "first-order",
+		Side:                "sell",
+		Symbol:              currency.Pair{Base: currency.BTC, Delimiter: currency.DashDelimiter, Quote: currency.ETH},
+		OrderType:           "market",
+		SelfTradePrevention: "CB",
+		Price:               1234,
+		Size:                0.0000001,
+	})
+	require.NoError(t, err)
+	assert.NotNil(t, result)
```

Comment:
Looks like a setup issue:
```
        	Error:      	Received unexpected error:
        	            	code: 126006 message: Request failed because the symbol {0} was not available
        	Test:       	TestPlaceMarginHFOrderTest
```